client/src/components/Navbar/Navbar.js

```diff
@@ -0,0 +1,18 @@
+import React from "react";
+import {Link} from "react-router-dom";
+
+class Navbar extends React.Component {
+    state = {
+    } 
+    render() { 
+        return (<div className="row justify-content-around" >
```

Comment:
These links need to be updated, but that can be done when those pages are coded.